config/kraft/README.md

```diff
@@ -114,9 +114,7 @@ We don't support any kind of upgrade right now, either to or from KRaft mode.  T
 
 Finally, the following Kafka features have not yet been fully implemented:
 
-* Support for certain security features: configuring a KRaft-based Authorizer, setting up SCRAM, delegation tokens, and so forth
-  (although note that you can use authorizers such as `kafka.security.authorizer.AclAuthorizer` with KRaft clusters, even
-  if they are ZooKeeper-based: simply define `authorizer.class.name` and configure the authorizer as you normally would).
+* Support for certain security features: setting up SCRAM, delegation tokens, and so forth.
 * Support for some configurations, like enabling unclean leader election by default or dynamically changing broker endpoints
```

Comment:
@jsancio did you fix the unclean leader election thing with your KIP?

Reply:
No. This is referring to supporting unclean leader election through configuration. Unclean leader election using the configuration `unclean.leader.election.enable` is not yet supported in KRaft. KRaft only supports unclean leader election through the `ElectLeaders` RPC.

KIP-704 was about propagating the fact that unclean leader election was performed to the topic partition replicas so they can recover from this. We plan to use this to recover from pending transactions.

Reply:
Thanks for clarifying.